tests/fixtures/git/github.com/demo/demo/demo.egg-info/PKG-INFO

```diff
@@ -6,7 +6,8 @@ Home-page: https://github.com/demo/demo
 Author: Sébastien Eustace
 Author-email: sebastien@eustace.io
 License: MIT
-Description: UNKNOWN
 Platform: UNKNOWN
-Provides-Extra: bar
 Provides-Extra: foo
+Provides-Extra: bar
+
```

Comment:
This is an invalid change that came along for the ride. Still need to find what test is causing this instability.

Reply:
ack, yeah not sure where that line came from, will remove